spec/models/stripe_wrapper_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+describe StripeWrapper do
+  describe StripeWrapper::Charge do
+    describe ".create" do
+      it "makes a successful charge", :vcr do
```

Comment:
You could push `:vcr` tag above, from `it` examples to `describe` level. Then all examples would have it automatically.